CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.0-beta.2 (2021-10-21)
+
+- Fixed the public API exports to use the proper lambda character. [#53](https://github.com/blackbaud/skyux-validation/pull/53)
```

Comment:
I wouldn't expose the "inner workings" to the changelog. Maybe something like, "Fixed the public exports API to be consistent with the other SKY UX component libraries."?

Reply:
Updated.